src/application/middlewares/auth.ts

```diff
@@ -40,6 +40,9 @@ export class AuthMiddleware implements Middleware {
       if (!payload) {
         return unauthorized(new InvalidTokenError('Invalid or expired token'))
       }
+      if (!payload.sub) {
```

Comment:
Why not use `if (!payload || !payload.sub)`?

Reply:
eslint complains if I use "||"

Reply:
Is there another simple way? What you think about add this config to eslint? So strange this sentence of ifs, It's just a simple OR statment.

Reply:
I don't quite understand why eslint prefers the "??" to be used instead of "||". But, it is possible to disable this rule if everyone agrees. And then I would put the verification in just an "if" and maybe create an "isValidPayload" variable to make it clearer when reading